kubectl-rook-ceph.sh

```diff
@@ -245,6 +249,21 @@ function run_rook_cr_status() {
   fi
 }
 
+function run_purge_osd() {
+  if [ "$#" -eq 1 ]; then
+    mon_endpoints=$($TOP_LEVEL_COMMAND --namespace "$ROOK_CLUSTER_NAMESPACE" get cm rook-ceph-mon-endpoints -o jsonpath='{.data.data}' | cut -d "," -f1)
+    ceph_secret=$($TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" exec deploy/rook-ceph-operator -- cat /var/lib/rook/"$ROOK_CLUSTER_NAMESPACE"/client.admin.keyring | grep "key" | awk '{print $3}')
+    $TOP_LEVEL_COMMAND --namespace "$ROOK_OPERATOR_NAMESPACE" exec deploy/rook-ceph-operator -- sh -c "export ROOK_MON_ENDPOINTS=$mon_endpoints \
+      ROOK_CEPH_USERNAME=client.admin \
+      ROOK_CEPH_SECRET=$ceph_secret \
+      ROOK_LOG_LEVEL=DEBUG \
```

Comment:
Is debug level needed? Or we can remove this line?

Reply:
no, it is not required, removed